scripts/constants.py

```diff
@@ -1,15 +1,22 @@
+import json
 import os
 import re
 from enum import Enum
 from pathlib import Path
 
 from dotenv import load_dotenv
+from eth_keys import keys
 from starknet_py.net.gateway_client import GatewayClient
 
+from scripts.artifacts import get_deployments
+
 load_dotenv()
 
 ETH_TOKEN_ADDRESS = 0x49D36570D4E46F48E99674BD3FCC84644DDD6B96F7C741B1562B82F9E004DC7
-EVM_ADDRESS = os.getenv("EVM_ADDRESS")
+EVM_PRIVATE_KEY = os.getenv("EVM_PRIVATE_KEY")
+EVM_ADDRESS = keys.private_key_to_public_key(
```

Comment:
fyi there is `private_key.public_key.to_checksum_address()` (maybe not in the current old version but we should be able to go back to the most recent one soon)